src/extension.ts

```diff
@@ -32,6 +33,19 @@ export function activate(context: vscode.ExtensionContext) {
 			void vscode.window.showTextDocument(doc)
 		}
 	}))
+
+	context.subscriptions.push(vscode.commands.registerCommand('vscode-flowr.session.connect', () => {
+		flowrSession.destroy()
+		flowrSession = new FlowrServerSession(outputChannel, diagnostics)
+	}))
+	context.subscriptions.push(vscode.commands.registerCommand('vscode-flowr.session.disconnect', () => {
+		if(flowrSession instanceof FlowrServerSession) {
+			flowrSession.destroy()
+			flowrSession = new FlowrInternalSession(outputChannel, diagnostics)
+		}
+	}))
+
+	context.subscriptions.push(new vscode.Disposable(()=> flowrSession.destroy()))
```

Comment:
```suggestion
	context.subscriptions.push(new vscode.Disposable(() => flowrSession.destroy()))
```

Why does this even pass linting :P